st2actions/st2actions/runners/fabricrunner.py

```diff
@@ -227,54 +227,3 @@ def _get_result_status(result, allow_partial_failure):
                 if not success:
                     return LIVEACTION_STATUS_FAILED
         return LIVEACTION_STATUS_SUCCEEDED if success else LIVEACTION_STATUS_FAILED
-
```

Comment:
Lol, yeah, I wanted to remove this many times before already but never done it :P